### PR TITLE
Fixes loading in duplicate JS & CSS files.

### PR DESCRIPTION
**Pull request by jeroendesloovere**
**Problem**

JS and CSS files should only be loaded in one time.

**Solution**

We added a check to see if the JS or CSS file is already added or not, so we don't load in duplicate files.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 Improvements:
 
 Bugfixes:
+* Frontend: no more duplicate JS-files.
```
Comment (WouterSioen) [inline CHANGELOG.md]: there should be a blank line above a list in markdown

File: src/Frontend/Core/Engine/Header.php [hunk 2]
```diff
@@ -151,18 +151,14 @@ public function addCSS($file, $minify = true, $addTimestamp = null)
...
 
-        $inArray = false;
-        foreach ($this->cssFiles as $row) {
-            if ($row['file'] == $file) {
-                $inArray = true;
-            }
-        }
+        $cssFile = array(
+            'file' => $file,
+            'add_timestamp' => $addTimestamp
```
Comment (WouterSioen) [inline src/Frontend/Core/Engine/Header.php]: multiline arrays should always contain trailing comma's

File: CHANGELOG.md (hunk 1 above)
Comment (jeroendesloovere) [inline CHANGELOG.md]: @WouterSioen Fixed

**Comment by WouterSioen** [comment]
:+1: will be merged

**Comment by jeroendesloovere** [comment]
@WouterSioen Should also be fixed in Backend I guess...

**Comment by WouterSioen** [comment]
Merged in 92ae4cb970bde032375534da29bca485b9519bce (due to conflicts)

**Pull request closed without merging**